packages/blade/docs/tokens/Motion.stories.mdx

```diff
@@ -0,0 +1,112 @@
+import { Meta, DocsContainer } from '@storybook/addon-docs';
+import { useTheme, BladeProvider } from '../../src//components';
+import { paymentTheme, bankingTheme } from '../../src//tokens';
+import styled from 'styled-components';
+import MovingDiv from '../components/MovingDiv';
+
+<Meta
+  title="Tokens/Motion"
+  parameters={{
+    docs: {
+      container: ({ children, context }) => {
+        const getThemeTokens = () => {
+          if (context.globals.themeTokens === 'paymentTheme') {
+            return paymentTheme;
+          }
+          if (context.globals.themeTokens === 'bankingTheme') {
+            return bankingTheme;
+          }
+          return null;
+        };
+        return (
+          <DocsContainer context={context}>
+            <BladeProvider
+              key={`${context.globals.themeTokens}-${context.globals.colorScheme}`}
+              themeTokens={getThemeTokens()}
+              colorScheme={context.globals.colorScheme}
+            >
+              {children}
+            </BladeProvider>
+          </DocsContainer>
+        );
+      },
+    },
+  }}
+/>
+
+# 🎬 Motion
+
```

Comment:
suggestion: in this only, can we add a section showing the usage of motion tokens in 1-2 components with styled components? that will make things easier for people to consumer